### PR TITLE
TCM-600 | Promos on Homepage

### PR DESCRIPTION
**Pull request by devender197**
### Summary

- [x] Promo added to home screen

### Issue

[TCM-600](https://hyfnla.atlassian.net/browse/TCM-600)

### Video/Screenshot

https://user-images.githubusercontent.com/24737954/122995520-9380a400-d35e-11eb-8eb1-a1cefd316d0f.mov

### CODE REVIEW

File: app/src/main/java/com/trx/consumer/managers/BackendManager.kt [hunk 1]
```diff
@@ -200,7 +200,8 @@ class BackendManager(private val api: BaseApi, private val cacheManager: CacheMa
 
     suspend fun promos(): ResponseModel {
         val path = EndpointModel.PROMOS.path
-        return call(RequestModel(endpoint = EndpointModel.PROMOS, path = path, params = null))
+        val params = cacheManager.accessToken()?.let { hashMapOf<String, Any>("jwt" to it) }
```
Comment (VivekV95) [inline app/src/main/java/com/trx/consumer/managers/BackendManager.kt:203]: Not sure this is needed, should just be able to add `PROMOS` to `isAuthenticated` in `EndpointModel`

**Review by VivekV95 — CHANGES_REQUESTED** [review]
Just one comment

File: app/src/main/java/com/trx/consumer/managers/BackendManager.kt (hunk 1 above)
Comment (devender197) [reply app/src/main/java/com/trx/consumer/managers/BackendManager.kt]: [Done](https://github.com/hyfn/trx-consumer-android/pull/129/commits/c1069a43a00277a7aa6dfe754b918daac64eefde#diff-96bdf8cd544df52ea2927bf8f74da8305c760cbb64aa464ce0438f526ac2e61cR74)

File: app/src/main/java/com/trx/consumer/models/common/EndpointModel.kt [hunk 2]
```diff
@@ -71,7 +71,7 @@ enum class EndpointModel {
     val isAuthenticated: Boolean
         get() {
             return when (this) {
-                AUTH,
+                AUTH,PROMOS,
```
Comment (onodude) [inline app/src/main/java/com/trx/consumer/models/common/EndpointModel.kt:74]: lets add after `PAYMENT_ADD` to keep the order

Comment (devender197) [reply app/src/main/java/com/trx/consumer/models/common/EndpointModel.kt]: [Done](https://github.com/hyfn/trx-consumer-android/pull/129/files#diff-96bdf8cd544df52ea2927bf8f74da8305c760cbb64aa464ce0438f526ac2e61cR83)

**Review by onodude — APPROVED** [review]
thanks!

**Pull request merged**